Makefile

```diff
@@ -2,10 +2,35 @@
 # Set env to override this value
 REGISTRY ?= openebs
 
+# Determine the arch/os
+ifeq (${XC_OS}, )
+  XC_OS:=$(shell go env GOOS)
+endif
+export XC_OS
+
+ifeq (${XC_ARCH}, )
+  XC_ARCH:=$(shell go env GOARCH)
+endif
+export XC_ARCH
+
+ARCH:=${XC_OS}_${XC_ARCH}
+export ARCH
+
+ifeq (${BASEIMAGE}, )
+ifeq ($(ARCH),linux_arm64)
+  BASEIMAGE:=arm64v8/ubuntu:18.04
+else
+	BASEIMAGE:=registry.access.redhat.com/ubi7/ubi-minimal:latest
```

Comment:
Can you change the base image to regular ubuntu. Accessing redhat may require some licensing changes.

Reply:
sounds good